### PR TITLE
[flutter_releases] Flutter beta 2.8.0-3.1.pre Framework Cherrypicks

### PR DESCRIPTION
**Pull request by godofredoc**
# Flutter beta 2.8.0-3.1.pre Framework

## Scheduled Cherrypicks

- commit: f4f23ecb5

### CODE REVIEW

**Comment by skia-gold** [comment]
Gold has detected about 2 new digest(s) on patchset 3.
View them at https://flutter-gold.skia.org/cl/github/93448

File: packages/flutter/lib/src/widgets/app.dart [hunk 1]
```diff
@@ -1668,15 +1669,15 @@ class _WidgetsAppState extends State<WidgetsApp> with WidgetsBindingObserver {
...
-            child: DefaultTextEditingActions(
+      child: SharedAppData(
+        child: Shortcuts(
+          debugLabel: '<Default WidgetsApp Shortcuts>',
+          shortcuts: widget.shortcuts ?? WidgetsApp.defaultShortcuts,
+          // DefaultTextEditingShortcuts is nested inside Shortcuts so that it can
+          // fall through to the defaultShortcuts.
+          child: DefaultTextEditingShortcuts(
+            child: Actions(
+              actions: widget.actions ?? WidgetsApp.defaultActions,
```
Comment (HansMuller) [inline packages/flutter/lib/src/widgets/app.dart:1680]: Line 1679: `child: DefaultTextEditingActions(`, should not have been dropped

Comment (godofredoc) [reply packages/flutter/lib/src/widgets/app.dart:1680]: Applied patch.

**Review by christopherfujino — APPROVED** [review]
LGTM

**Comment by godofredoc** [comment]
Landing with Mac module_test_ios, the fix for the test is landing in https://github.com/flutter/flutter/pull/93556

**Pull request merged**